src/source/lexbase.d

```diff
@@ -564,6 +564,25 @@ unittest {
 		assert(t.error.toString(context) == error);
 	}
 
+	auto checkIncompleteInput(string s, TokenType expected) {
```

Comment:
I'm not sure why we are passing something called `expected` here, as this is absolutely not what we are expecting in any way shape of form. What we are expecting is an exception to be thrown.